### PR TITLE
Cleanup warnings in the code on current rust version.

### PR DESCRIPTION
**Pull request by kraythen**
I just removed some warnings in the code.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
It has nothing to do with warnings. :-/

**Comment by kraythen** [comment]
Sorry I forgot to PUSH the actual warnings changes. :*) Done now.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ If a project depends on this version, don't forget to add in your `Cargo.toml`:
 v2 = ["GSL/v2"]
 ```
 
+If you always want to run the v2 features you can, instead, use the following in your `Cargo.toml`:
```
Comment (GuillaumeGomez) [inline README.md]: Nice one! :)

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by GuillaumeGomez** [comment]
I published `1.0`. Don't hesitate to ask if you want/need something else. :)

**Pull request merged**